CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used. 
+
+# Pull Requests
+
+Pull requests should be named with the format `crate: short description of change`, and should use
+lower case letters. Always make a pull request from a new branch, that is named similarly, but with
+only a few words: `crate-short-description`. If adding a feature or enhancement, use the term `add`
+or something sufficiently similar. If fixing a bug, use the term `fix`, or something sufficiently
+similar. Avoid force-pushing to a pull request branch, as this erases review comment history.
```

Comment:
Its probably worth noting here that the default policy is to squash on merge, otherwise this will appear quite odd that we encourage tons of commits.

Reply:
That is noted in the section about merging

Reply:
(Probably by linking to the `# Merging` section since its listed there)